.changeset/modern-ways-develop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/node': patch
+---
+
+Cancel response stream when connection closes
```

Comment:
```suggestion
The node adapter now automatically cancels the `ReadableStream` of your streaming responses when the visitor disconnects, allowing you to gracefully clean up resources.
```
Just making it more descriptive. LMK if this looks helpful.

Reply:
eh, I personally find the original descriptive enough (when compared to other changesets). Gonna defer to the maintainers, I don't have a strong preference

Reply:
I don't really have a hard preference either, but since this is only a patch, I think a short description is also fine by me.